lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -293,7 +313,9 @@ <h3 class="course-title" id="course-title-${enrollment.course_id}">
               entitlementUUID: '${ entitlement.course_uuid | n, js_escaped_string }',
               currentSessionId: '${ entitlement_session.course_id if entitlement_session else "" | n, js_escaped_string }',
               enrollUrl: '${ reverse('entitlements_api:v1:enrollments', args=[str(entitlement.uuid)]) | n, js_escaped_string }',
-              courseHomeUrl: '${ course_target | n, js_escaped_string }'
+              courseHomeUrl: '${ course_target | n, js_escaped_string }',
+              expiredAt: '${ entitlement.expired_at_datetime | n, js_escaped_string }',
```

Comment:
Should probably use entitlement_expired_at (or drop that var and use entitlement.expired_at_datetime everywhere).

Reply:
Actually, this is inside a block where we know entitlement_expired_at is None anyway...

Reply:
Oh, OK.  My first comment is garbage because entitlement_expired_at is a localized string time, but we want the raw datetime here.

But second comment might stand.  Maybe we don't need this line.  But better safe than sorry in case this block gets moved.  So leaving in is good too.